### PR TITLE
chore: 6.10.2 dev prep + cleanup

### PR DESCRIPTION
**Pull request by brainbicycle**
The type of this PR is: **chore**

### Description

- Updates version number
- Removes some docs mentioning updating changelog
- Updates fastlane + sentry plugin
- Renames old changelog and adds a comment pointing to new process

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

- 

#### Android user-facing changes

-

#### Dev changes

- Prepare for 6.10.2 development

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  The CHANGELOG is not valid YML:
Error: ENOENT: no such file or directory, open 'CHANGELOG.yml'
    at Object.openSync (fs.js:474:3)
    at Object.readFileSync (fs.js:375:35)
    at dangerfile.ts:126:33
    at Generator.next (<anonymous>)
    at dangerfile.ts:27:71
    at new Promise (<anonymous>)
    at __awaiter (dangerfile.ts:23:12)
    at validateChangelogYMLFile (dangerfile.ts:123:40)
    at dangerfile.ts:185:15
    at Generator.next (<anonymous>)
  </td>
    </tr>
  </tbody>
</table>



### This PR contains the following changes:

- Dev changes (Prepare for 6.10.2 development)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ed0733dbd9b2c871b4b84f0cf52a91f823e9a073
</p>

**Review by pvinis — APPROVED** [review]
Nice. Probably we need to remove https://github.com/artsy/eigen/blob/brian/6.10.2-dev-prep/dangerfile.ts#L92-L117 this part too.

**Comment by pvinis** [comment]
Could you also remove `"yaml"` from `package.json`? Seems like that file was the only use.

File: fastlane/Fastfile [hunk 1]
```diff
@@ -104,9 +96,9 @@ end
 date_str = DateTime.now.strftime("%Y.%m.%d.%H")
 lane :set_build_version_ios do
   system("/usr/libexec/PlistBuddy -c \"Set CFBundleShortVersionString #{app_json['version']}\" #{app_plist}")
-  system("/usr/libexec/PlistBuddy -c \"Set CFBundleShortVersionString #{app_json['version']}\" #{sticker_plist}")
+  system("/usr/libexec/PlistBuddy -c \"Set CFBundleShortVersionString #{app_json['version']}\" \"#{sticker_plist}\"")
   system("/usr/libexec/PlistBuddy -c \"Set CFBundleVersion #{date_str}\" #{app_plist}")
-  system("/usr/libexec/PlistBuddy -c \"Set CFBundleVersion #{date_str}\" #{sticker_plist}")
+  system("/usr/libexec/PlistBuddy -c \"Set CFBundleVersion #{date_str}\" \"#{sticker_plist}\"")
```
Comment (pvinis) [inline fastlane/Fastfile:101]: were these wrong?

Comment (brainbicycle) [reply fastlane/Fastfile:101]: The stickers one was failing because there was a space in the filepath so just had to wrap in quotes

**Pull request merged**